### PR TITLE
Additions to get RPi3/RPi4 to build again

### PR DESCRIPTION
**Pull request by Herrie82**
Initial work to get RPi3/RPi4 building again.

UI works, WAM works, WiFi & BT work, so good enough to start with.

### CODE REVIEW

**Comment by Herrie82** [comment]
@shr-project Only issue I'm getting is the following while gzipping the image:
gzip: skipping: luneos-dev-image-raspberrypi3-64.rootfs-unstable-20240220152351.rpi-sdimg does not exist

Seems related to: https://github.com/webosose/meta-webosose/commit/c8d1433b6f62c8c67a934fa36d1aca6e30ca6e42

File: meta-luneos/conf/distro/luneos.conf [hunk 1]
```diff
@@ -13,3 +13,17 @@ MAINTAINER = "Simon Busch <morphis@gravedo.de>"
...
+    commercial_gstreamer1.0-omx \
+    commercial_gstreamer1.0-libav \
+    commercial_lame \
+    commercial_libav \
+    commercial_libomxil \
+    commercial_mpg123 \
+    commercial_mpeg2dec \
+    commercial_x264 \
+    synaptics-killswitch \
+"
```
Comment (shr-project) [inline meta-luneos/conf/distro/luneos.conf:29]: Why do you think this is needed when we have:

meta-luneos/conf/distro/include/luneos.inc:LICENSE_FLAGS_ACCEPTED = "commercial"
?

Comment (Herrie82) [reply meta-luneos/conf/distro/luneos.conf:29]: Weird it didn't behave before, does seem to behave now without these. Will drop them

Comment (Herrie82) [reply meta-luneos/conf/distro/luneos.conf]: The synaptics one is needed, so added it back. Rest is gone

**Comment by Herrie82** [comment]
@shr-project This builds & boots now, it has a few bugs still, but that could be due to tweaking. It builds, boots to UI, network works, wifi works, so good enough to merge I'd say. @ericblade tested :)

**Review by shr-project — APPROVED** [review]

**Pull request merged**